src/core/eme/session_events_listener.ts

```diff
@@ -322,8 +323,8 @@ function handleKeyStatusesChangeEvent(
       throw err;
     })
   );
-  return observableConcat(getKeyStatusesEvents(session, keySystemOptions, keySystem),
-                          callback$);
+  return observableMerge(getKeyStatusesEvents(session, keySystemOptions, keySystem),
```

Comment:
Technically, there's no difference here as everything from the first Observable are synchronous.

But I felt this should be more of a merge.